docs/setup-cluster/slurm/slurm-known-issues.rst

```diff
@@ -381,6 +381,10 @@ Some constraints are due to differences in behavior between Docker and Singulari
  PBS Known Issues
 ******************
 
+-  If the ``Cluster`` tab on the DeterminedAI WebUI does not display the GPU information, there
```

Comment:
Is it `DeterminedAI` or `Determined AI`?

Is `WebUI` the appropriate wording to use?

Reply:
I removed `DeterminedAI` as per Tara's suggestion. I think `WebUI` is fine here. Please let me know why you think it might be an inappropriate word. I can change it if needed.